test/test_export.py

```diff
@@ -472,3 +472,15 @@ def func(a, b, c):
         dynamo_result = out_graph(*flat_input)
 
         self.assertTrue(torchdynamo.utils.same(real_result, dynamo_result))
+
+    def test_min_op(self):
```

Comment:
This is vague, as it uses export as a black box to enforce a guarantee. Instead, check out how our other test are written, in test_misc for example.